### ISSUE TITLE
Add a Bunis HSPC data getter

### ISSUE THREAD

**Issue by LTLA** [LTLA/scRNAseq#28]
@dtm2451 the time has cometh. Should be as simple as copying one of the other getters to `BunisHSPCData.R`, renaming the function and changing the path to `bunis-hspc`. 

- Make sure you set `rowdata=TRUE` in `.create_sce()` to get the row data.
- Add yourself to `Authors@R`.

**Comment by LTLA** [comment]
So, uh, Dr Bunis. Any plans for this, or what?

**Comment by dtm2451** [comment]
Yes yes!  Fell off my radar, but it's back on for this week.  I'll put something together by the weekend.

**Comment by dtm2451** [comment]
I'm not sure of the best ways to convert from dgT to dgC, but the counts matrix is already dgT right out of https://github.com/LTLA/scRNAseq/blob/a04b20d9e387afdd8da27ea263cd42b35cf2f997/inst/scripts/2.6.0/make-bunis-hspc-data.Rmd#L29.

Might as well add the conversion right around there, no?

**Comment by LTLA** [comment]
Oh crap, didn't notice that's where it was coming from. Yes, the conversion would be done there. I can do that myself.
